datadog_checks_downloader/datadog_checks/downloader/cli.py

```diff
@@ -23,6 +24,34 @@ def __is_canonical(version):
     return re.match(P, version) is not None
 
 
+def __find_shipped_integrations():
```

Comment:
Could you briefly comment the logic in this function? It's not immediately clear...